src/schema/schema.graphql

```diff
@@ -1846,6 +1856,12 @@ type Query {
 	preferences: Preferences
 	relatedContacts(email: String!): RelatedContacts
 	shareInfos(addresses: [String!]): [[ShareInfo]]
+	recoverAccount(
```

Comment:
I think this fits better as a `mutation`, since it is un-cacheable and we're altering something on the back end. As a mutation, we can `catch` the error that happens when the mutation fails and we can remove the `catch` statement on the `ZimbraBatchClient::recoverAccount` method.